tests/foreman/ui_airgun/test_discoveredhost.py

```diff
@@ -0,0 +1,496 @@
+    "proxy.type=proxy" in PXE default in unattended mode.
+
+    :id: fc13167f-6fa0-4fe5-8584-7716292866ce
+
+    :Setup: Provisioning should be configured
+
+    :Steps: Boot a host/VM using modified discovery ISO.
+
+    :expectedresults: Host should be successfully discovered
+
+    :CaseLevel: System
+    """
+    with LibvirtGuest(boot_iso=True) as pxe_less_host:
+        host_name = pxe_less_host.guest_name
+        with session:
+            session.organization.select(org_name=module_org.name)
+            session.location.select(loc_name=module_loc.name)
+            discovered_host_values = session.discoveredhosts.wait_for_entity(
+                host_name)
+            assert discovered_host_values['Name'] == host_name
+
+
+@run_in_one_thread
+@run_only_on('sat')
+@skip_if_not_set('compute_resources', 'vlan_networking')
+@tier3
+@upgrade
+def test_positive_provision_using_quick_host_button(
+        session, module_org, module_loc, provisioning_env):
+    """Associate hostgroup while provisioning a discovered host from
+    host properties model window and select quick host.
+
+    :id: 34c1e9ea-f210-4a1e-aead-421eb962643b
+
+    :Setup:
+
+        1. Host should already be discovered
+        2. Hostgroup should already be created with all required entities.
+
+    :expectedresults: Host should be quickly provisioned and entry from
+        discovered host should be auto removed.
+
+    :CaseLevel: System
+    """
+    with LibvirtGuest() as pxe_host:
+        host_name = pxe_host.guest_name
+        with session:
+            session.organization.select(org_name=module_org.name)
+            session.location.select(loc_name=module_loc.name)
+            discovered_host_values = session.discoveredhosts.wait_for_entity(
+                host_name)
+            assert discovered_host_values['Name'] == host_name
+            session.discoveredhosts.provision(
+                host_name,
+                provisioning_env['host_group'],
+                module_org.name,
+                module_loc.name,
+            )
+            pxe_host_name = '{0}.{1}'.format(
+                host_name, provisioning_env['domain'])
+            # After this step we will be redirected to the created host details
+            # view.
+            values = session.host.get_details(pxe_host_name)
+            assert values['properties']['properties_table']['Status'] == 'OK'
+            # Check that provisioned host is not in the list of discovered
+            # hosts anymore
+            assert not session.discoveredhosts.search(
+                'name = {0}'.format(host_name))
+
+
+@run_in_one_thread
+@run_only_on('sat')
+@skip_if_not_set('compute_resources', 'vlan_networking')
+@tier3
+def test_positive_update_name(
+        session, module_org, module_loc, provisioning_env):
+    """Update the discovered host name and provision it
+
+    :id: 3770b007-5006-4815-ae03-fbd330aad304
+
+    :Setup: Host should already be discovered
```

Comment:
not a blocker - in order to honor what docstring says, we should have the discovered host already prepared by some fixture (setup).
this applies to all ocurrences of this